src/table/settings.ini

```diff
@@ -560,6 +560,14 @@ strval   = STR_CONFIG_SETTING_SIGNALSIDE_LEFT
 proc     = RedrawScreen
 cat      = SC_BASIC
 
+[SDT_BOOL]
+base     = GameSettings
+var      = construction.auto_remove_signals
+flags    = SLF_NOT_IN_SAVE | SLF_NO_NETWORK_SYNC
+def      = false
+str      = STR_CONFIG_SETTING_AUTO_REMOVE_SIGNALS
+strhelp  = STR_CONFIG_SETTING_AUTO_REMOVE_SIGNALS_HELPTEXT
+
```

Comment:
If this is intended to be a client-only setting (i.e. using `SLF_NOT_IN_SAVE | SLF_NO_NETWORK_SYNC`), it should go in ClientSettings/GUISettings.

Otherwise if it is intended to be a company setting (as suggested by the placement in the settings GUI), it should go in CompanySettings/company_settings.ini, and `SLF_NOT_IN_SAVE | SLF_NO_NETWORK_SYNC` should be removed. In this case you would not need to pass it though DoCommand parameters but could look it up from _current_company.

Reply:
We would prefer this as a client-setting, as it is how you like to build. And very nicely spotted @JGRennison :)

Reply:
It is indeed intended as a client setting. Looking at the rest of the settings, ClientSettings/GUI indeed looks like a more logical place to put it, I'll change it 👍